.github/workflows/cicd.yml

```diff
@@ -114,22 +114,18 @@ jobs:
       - name: build library
         run: pnpm run build
 
-      - name: lerna version
+      - name: Publish to GitHub repository
```

Comment:
Misschien moeten we dit gewoon "Run Changesets" noemen.

Reply:
Ik denk dat het inderdaad nog iets duidelijker kan, maar "Run Changesets" is alleen duidelijk voor mensen die toch wel snappen wat er gebeurt als ze de code lezen — nogal technisch 😅